requirements.txt

```diff
@@ -1,6 +1,7 @@
 # migrid core dependencies on a format suitable for pip install as described on
 # https://pip.pypa.io/en/stable/reference/requirement-specifiers/
 future
+python-openid2
```

Comment:
We need to investigate if this one **just** works with py2. I can see we use `python-openid` for py2 but `python-openid2` for py3 in `docker-migrid`.

Reply:
Interesting - I definitely think the preference would be to get to a dependency that "juts works" but: given a large push in this PR was to be as minimally behaviour affecting as possible, I'm actually tempted to ensure that here we go with exactly what's running _right now_ and then we could combine this later. What do you think?

Reply:
Yes, on pypi the maintainers of python-openid and python-openid2 seem to overlap, so the latter may also just work - needs testing, though.
Until then the safest bet is probably to split it into a `python-openid` for `python_version<"3"` and `python-openid2`  for `python_version>="3"`. 
Including a line or two of comment about the situation would be even better. It can point to `Dockerfile.X` from  `docker-migrid` for details.
For the record the situation is actually even more complicated since we use a third `python3-openid` fork of the original through the identically named distro package on Rocky9. We can e.g. point to the `python3-openid` package with `python_version>="3.7"` in requirements to reflect that.